stdlib/universe/difference.go

```diff
@@ -344,8 +371,11 @@ func (d *difference) updateInt(v int64, valid bool) (int64, bool) {
 		d.valid = true
```

Comment:
I think this is the line where you'll need to add the initial zero logic for the first point. I think you can just change this to:

```
if !d.valid {
    d.valid = true
    return 0, d.initialZero
}
```

If `keepFirst` is false, the true/false value of this doesn't matter. If `keepFirst` is true, then it will see true and append the integer as the return value.

I think it's ok to leave the beginning of this function the way that it is and not change it to include initial zero. I think that's consistent with the current behavior which is that seeing a null will result in a null value being appended.